### DISCUSSION TITLE
How to interpret the output importance scores of GNNexplainer

### DISCUSSION THREAD

**Discussion by cduan8** [pyg-team/pytorch_geometric#9732] [Q&A]
Hi PyG team and community,

I am recently using the GNNexplainer in geometric.explain to analyze the feature importance of a trained GAT model, and having some technical questions regarding the interpretation I can derive from the results. 
1. When I choose "phenomena" and "attributes", to find the overall importance of each feature, should I sum the node score mask over all the nodes? 
2. After getting the importance score for each feature, how should one determine whether a feature is important or not? Is there a general rule to follow to characterize a feature as unimportant for the overall performance?
3. Is there any other physical metrics which we can convert the importance scores to? (Such as influence on the loss function values)

Thank you for your help in advance!
